CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.42.0
+* Add some tests and verify basic support for triple-shift, and check
+  to be sure that using the function as type parameter feature does not crash
+  dartdoc (#2518, #2517, #2603).
+* BREAKING CHANGE: Refactor of dartdoc features and annotations.  Use
+  `hasFeatures` in templates in place of checking `featuresAsString` as a
+  boolean (#2600).
+* Implement generic type parameters on annotations. (#2600, #2516).
+* No longer depend on mustache. (#2596)
```

Comment:
👍 

We might be able to remove the package:mustache dep in the sdk after this rolls in.

Reply:
good point!  I'll take a look at that.